pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -2,67 +2,11 @@ package v1
 
 import (
 	"fmt"
-	"sort"
 
-	ign "github.com/coreos/ignition/config/v2_2"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-// MergeMachineConfigs combines multiple machineconfig objects into one object.
-// It sorts all the configs in increasing order of their name.
-// It uses the Ignition config from first object as base and appends all the rest.
-// Kernel arguments are concatenated.
-// It uses only the OSImageURL provided by the CVO and ignores any MC provided OSImageURL.
-func MergeMachineConfigs(configs []*MachineConfig, osImageURL string) *MachineConfig {
```

Comment:
Does the `pkg/apis` name mean this subpackage is meant to be consumed by other repositories? If so, is it safe to remove this?

(I can’t see any user inside GitHub/openshift , at least).

Reply:
The apis package will be moved into `openshift/api` eventually (hence moving out this func which needs ignition types). It's used by `installer` afaik